addons/binding/org.openhab.binding.foobot/README.md

```diff
@@ -0,0 +1,78 @@
+# Foobot Binding
+
+This binding fetches the Indoor Air Quality data of your Foobot device from the Foobot cloud service.
+
+To use this binding, you first need to [register and get your API key](https://api.foobot.io/apidoc/index.html).
+
+## Supported Things
+
+There is exactly one supported thing type, which provides the Indoor air quality information from the foobot device.
+It has the `foobot` id.
+If you have multiple foobots in your home or office, you can add multiple Things, using the unique MAC Address for each device.
+
+## Discovery
+
+There is no discovery implemented.
+You have to create your things manually.
+
+## Binding Configuration
+
+The binding has no configuration options, all configuration is done at Thing level.
+
+## Thing Configuration
+
+The thing has a few configuration parameters:
+
+| Parameter | Description                                                             |
+|-----------|-------------------------------------------------------------------------|
+| apikey    | API Key from https://api.foobot.io/apidoc/index.html. Mandatory.        |
+| username  | your UserName for the foobot App. Mandatory.                            |
+| MAC Id    | Unique ID of your foobot device. Mandatory.                             |
+| refresh   | Refresh interval in minutes. Optional, the default value is 30 minutes. |
```

Comment:
How often is data pushed from the phyisical device to the cloud, I assume its not uploaded in realtime, what is the minimum resolution which makes sense. Do you nee a guard in place to protect that all people will configure 1 min?